WindowsClient/_Data/_Items/Console.xaml.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace WindowsClient._Data._Items
+{
+    /// <summary>
+    /// Interaction logic for Console.xaml
+    /// </summary>
+    public partial class Console : UserControl
```

Comment:
what does this class do?

Reply:
write log warnings in console, but it's not ready.